toolkit/tools/grapher/grapher.go

```diff
@@ -25,6 +26,20 @@ var (
 	strictGoals      = app.Flag("strict-goals", "Don't allow missing goal packages").Bool()
 	strictUnresolved = app.Flag("strict-unresolved", "Don't allow missing unresolved packages").Bool()
 
+	resolveCyles		 = app.Flag("resolve-cycles-from-repo", "Let grapher resolve cycles by marking rpms available in repo as remote").Bool()
+	outDir      = exe.OutputDirFlag(app, "Directory to download packages into.")
+        existingRpmsDir          = app.Flag("rpm-dir", "Directory that contains already built RPMs. Should contain top level directories for architecture.").Required().ExistingDir()
+        existingToolchainRpmDir = app.Flag("toolchain-rpms-dir", "Directory that contains already built toolchain RPMs. Should contain top level directories for architecture.").Required().ExistingDir()
+        tmpDir                  = app.Flag("tmp-dir", "Directory to store temporary files while downloading.").String()
+
+        workerTar            = app.Flag("tdnf-worker", "Full path to worker_chroot.tar.gz").Required().ExistingFile()
+        repoFiles            = app.Flag("repo-file", "Full path to a repo file").Required().ExistingFiles()
+        usePreviewRepo       = app.Flag("use-preview-repo", "Pull packages from the upstream preview repo").Bool()
+        disableUpstreamRepos = app.Flag("disable-upstream-repos", "Disables pulling packages from upstream repos").Bool()
+        toolchainManifest    = app.Flag("toolchain-manifest", "Path to a list of RPMs which are created by the toolchain. Will mark RPMs from this list as prebuilt.").ExistingFile()
+	tlsClientCert = app.Flag("tls-cert", "TLS client certificate to use when downloading files.").String()
+	tlsClientKey  = app.Flag("tls-key", "TLS client key to use when downloading files.").String()
```

Comment:
The TLS cert/key files are something we don't use anymore and can drop it here.

Reply:
dropped tlsKey and cert

Reply:
FYI there may be customer demand to use the certs and keys again, would prefer to preserve these everywhere to maintain consistency.

Reply:
added back certs and keys